tutorials/singularity-containers-with-cloud-build/README.md

```diff
@@ -5,152 +5,309 @@ author: wardharold
 date_published: 2019-01-18
 ---
+
 This tutorial shows you how to use [Cloud Build](https://cloud.google.com/cloud-build/) to build [Singularity](https://www.sylabs.io/singularity/) containers. 
-In constrast to [Docker](https://www.docker.com/), the Singularity container mechanism is designed specifically for HPC workloads. 
+In constrast to [Docker](https://www.docker.com/), the Singularity container binary is designed specifically for HPC workloads. 
 
 [![button](http://gstatic.com/cloudssh/images/open-btn.png)](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/community&page=editor&tutorial=tutorials/singularity-containers-with-cloud-build/tutorial.md)
 
-## (OPTIONAL) Create a project with a billing account attached 
-**(you can also use an existing project and skip to the next step)**
+### How do I launch jobs on the cloud?
+
+If you are traditionally an HPC user, you are most likely familiar with working
+on a command line, and specifically launching jobs on a [SLURM](https://slurm.schedmd.com/) or 
+[Sun Grid Engine](http://gridscheduler.sourceforge.net/) cluster with `qsub` or `sbatch`.
+The cloud equivalent of that is [gcloud](https://cloud.google.com/sdk/install). With gcloud you
+can manage and interact with resources on Google Cloud, and you don't need to leave your terminal.
+
+If you haven't yet, be sure that you have [installed gcloud](https://cloud.google.com/sdk/install)
+on your machine to interact with Google Cloud. You can next proceed by either creating a new project, or using an already existing one.
+
+## Project Setup
+
+The first sections here will help you to set up your project and working space.
+This really only needs to be done once. If you already have done these steps,
+you can source your env.sh and move on to creating [Singularity Builds](#singulrity-build)
 
-### Setup project related environment variables
+### Step 1: Project environment variables
+
+For this first step, you need to define some important variables.
 Edit the file env.sh and replace
+
 * [YOUR_ORG] with the name of the organization that will own your project
 * [YOUR_BILLING_ACCOUNT_NAME] with the name of the account responsible for any costs associated with your project
 * [NAME FOR THE PROJECT YOU WILL CREATE] with the name of your project
 * [COMPUTE ZONE YOU WANT TO USE] with the name of the Cloud Platform compute zone that will contain your project
 
+The Organization and billing account should already exist (if you need to create an organization,
+see the [documentation here](https://cloud.google.com/resource-manager/docs/creating-managing-organization)), and if you are creating a new project, you can derive the the project name and zone right now! Otherwise,
+you can fill in an existing project name.
+
+**How do I find these variables on the command line?**
+
+If you already have gcloud installed and an organization and/or project in mind, it can be a hassle to
```

Comment:
If you already have gcloud installed and an organization and/or project in mind ~~, it can be a hassle to
go to a web browser and click around. H~~here is how you can look up your organization,
billing account name, project, and zone:
---
I like it but the tech editors would hate it. :(